docs/src/pages/demos/expansion-panels/ControlledExpansionPanels.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    heading: {
+      fontSize: theme.typography.pxToRem(15),
+      flexBasis: '33.33%',
+      flexShrink: 0,
+    },
+    secondaryHeading: {
+      fontSize: theme.typography.pxToRem(15),
+      color: theme.palette.text.secondary,
+    },
+  }),
+);
+
+function ControlledExpansionPanels() {
+  const classes = useStyles();
+  const [expanded, setExpanded] = React.useState<string | false | null>(null);
```

Comment:
```suggestion
  const [expanded, setExpanded] = React.useState<string | false>(false);
```
Is there something special about `null`?

Reply:
👍